.idea/Simulator2020.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you remove all `.idea/` files from this PR?

Reply:
Give me a second

Reply:
Done